### PR TITLE
Pia 3561 allow injecting a bottom navigation bar ra

### PR DESCRIPTION
**Pull request by vizaknaidavid**
bottom nav

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingNavigationBarBottomAdapter.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- note: Bottom navigation only.
+*/
+public protocol DigitalInvoiceOnboardingNavigationBarBottomAdapter: InjectedViewAdapter {
+    /**
+     *  Set the callback for the continue button action.
+     *
+     * - Parameter callback: An  action callback, which should be retained and called in get started button action method
+     */
+    func setContinueButtonClickedActionCallback(_ callback: @escaping  () -> Void)
```
Comment (a-szotyori) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingNavigationBarBottomAdapter.swift:22]: [Request] There was an inconsistency in the designs in Figma. The button title "Continue" was wrong. It should have been "Get started". I fixed the [designs](https://www.figma.com/file/m4OtCvxWxPMeB0Z5k4Aeow/Gini-Photo-Payment-%2F-iOS?node-id=1094%3A21729&t=fzl3VYsWSuD7JlHL-0) and I need to ask you to rename this method to `setGetStartedButtonClickedActionCallback()` and change the button title to "Get started"/"Los geht's". Sorry 🙈

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Small change needed because the design in Figma was wrong...

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingNavigationBarBottomAdapter.swift (hunk 1 above)
Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingNavigationBarBottomAdapter.swift]: Fixed :D

image

**Review by a-szotyori — APPROVED** [review]
Thank you!

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**